pkg/constants/constants.go

```diff
@@ -52,4 +52,7 @@ const (
 
 	// ControlInfoMsgKey - control info message key.
 	ControlInfoMsgKey = "ControlInfo"
+
+	// hubOfHubsLocalResource - local-resource label
+	HubOfHubsLocalResource = "hub-of-hubs.open-cluster-management.io/local-resource"
```

Comment:
I would name the const `HubOfHubsLocalResourceAnnotation` to avoid confusion like happened in this PR (it was used with labels filter predicate)